config.yaml

```diff
@@ -0,0 +1,28 @@
+torch:
+  device: 'cuda'
+  seed: 42 # For global seed
+
+loader:
+  path_to_logos_PURE: "data/all_data.json"
+  max_data_size: -1
+
+processor:
+  batch_size: 1028
+  test_ratio: 0.3
+
+nn_optimizer:
+  epochs: 1
+  batch_size: 256
+  shuffle: true
+  optimizer: adam
+  learning_rate: 0.01
+  loss: binary_crossentropy
+
+nn_model:
+  embedding_dim: 100
+  hidden_dim: 128
+  num_layers: 2
+
+trainer:
+  run_name: mock_run
+  on_cluster: False
```

Comment:
interesting, i have to ask about this - on_cluster

Reply:
It simply just disable `mlflow` in `metrics.py` cause i dont think it works on cluster

Reply:
ahh makes sense